kimchi/src/constants.rs

```diff
@@ -0,0 +1,12 @@
+/// Number of constraints produced by the permutation argument.
+pub const PERM_CONSTRAINTS: u32 = 3;
```

Comment:
the PERM constants are actually specific to the permutation only, so I don't think they should be moved out of the permutation file : o

Reply:
Done https://github.com/o1-labs/proof-systems/pull/974/commits/91350343175bf874d16ada51a0ee486a6135db73